test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -37,8 +38,12 @@ def assert_grad(
 
         inputs_ = []
         for i in inputs:
-            i.requires_grad = True
-            inputs_.append(i.to(dtype=torch.float64, device=self.device))
+            if torch.is_tensor(i):
+                i = i.to(
+                    dtype=torch.cdouble if i.is_complex() else torch.double,
```

Comment:
please add a comment to explain that we need this because we want to run gradcheck and gradcheck with double and cdouble